x-pack/plugins/case/common/api/cases/case.ts

```diff
@@ -15,12 +15,23 @@ import { CaseConnectorRt, ESCaseConnector, ConnectorPartialFieldsRt } from '../c
 // eslint-disable-next-line @kbn/eslint/no-restricted-paths
 export { ActionTypeExecutorResult } from '../../../../actions/server/types';
 
-const StatusRt = rt.union([rt.literal('open'), rt.literal('closed')]);
+const CaseStatusRt = rt.union([
```

Comment:
I'm not sure that it makes an actual difference, but it seems fun to do it like this:
```
export enum CaseStatuses {
  open = 'open',
  'in-progress' = 'in-progress',
  closed = 'closed',
}
const CaseStatusRt = rt.union([
  rt.literal(CaseStatuses.open),
  rt.literal(CaseStatuses['in-progress']),
  rt.literal(CaseStatuses.closed),
]);
```

Reply:
I think if you do this, you can get rid of the `export type CaseStatus = rt.TypeOf<typeof CaseStatusRt>;` and use `CaseStatuses` everywhere. I think even if you don't use this you could replace `CaseStatus` with `CaseStatuses` everywhere